jleeboyd/family-bank-app
# Transaction history + account balance update

### User Story
This PR provides functionality for a working transaction history, and updateable account balance.

### Context / Additional Information

**To test**
1. Create a new account
2. Click the newly created account
3. Deposit an amount
4. See deposited amount added to account balance.

5. Withdraw an amount
6. See amount taken from account balance total.

### Acceptance Criteria
- [x] - Account balance updates accordingly with Deposit/Withdraw
- [x] - Transaction history working with recycler view.
